.gitlab-ci.yml

```diff
@@ -8,6 +8,8 @@ reference:
     - update-alternatives --set iptables /usr/sbin/iptables-legacy
   script:
     - buildah build --file build.Containerfile --output build .
+  after_script:
+    - sha256sum build/core/* build/wallettool/bin/*  build/wallettool/lib/*
```

Comment:
If we want to recursively shasum the entire build directory, we could use

`find build/ -exec sha256sum {} +`

Reply:
I would do that but because we want to compare that line with the lines in the non-reference builds, I think it's better to enumerate the paths.